### PR TITLE
Allow to override variables in install-bootstrap-creds.sh

### PR DESCRIPTION
**Pull request by sathieu**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

**What is the new behavior (if this is a feature change)?**

**Other information**:

### CODE REVIEW

File: deploy/install-bootstrap-creds.sh [hunk 1]
```diff
@@ -17,9 +17,9 @@ set -eu
 
 # fill out these variables if you want to change the
 # default pgo bootstrap user and role
-PGOADMIN_USERNAME=admin
-PGOADMIN_PASSWORD=examplepassword
-PGOADMIN_ROLENAME=pgoadmin
+PGOADMIN_USERNAME="${PGOADMIN_USERNAME-admin}"
+PGOADMIN_PASSWORD="${PGOADMIN_PASSWORD-examplepassword}"
+PGOADMIN_ROLENAME="${PGOADMIN_ROLENAME-pgoadmin}"
```
Comment (jkatz) [inline deploy/install-bootstrap-creds.sh:22]:
```suggestion
PGOADMIN_USERNAME="${PGOADMIN_USERNAME:-admin}"
PGOADMIN_PASSWORD="${PGOADMIN_PASSWORD:-examplepassword}"
PGOADMIN_ROLENAME="${PGOADMIN_ROLENAME:-pgoadmin}"
```

**Review by jkatz — CHANGES_REQUESTED** [review]

**Comment by sathieu** [comment]
Back to you @jkatz

**Review by jkatz — APPROVED** [review]

**Comment by jkatz** [comment]
LGTM :+1:

**Comment by jkatz** [comment]
Committed. Backpatched down to `REL_4_5`

**Pull request merged**